charms/pipelines-ui/src/charm.py

```diff
@@ -93,6 +93,44 @@ def set_pod_spec(self, event):
 
         healthz = f"http://localhost:{config['http-port']}/apis/v1beta1/healthz"
 
+        env = {
```

Comment:
I feel like some of these values should be juju config options for further configurability, but I might be missing context